### PR TITLE
politeness logistic code: data format

### PR DESCRIPTION
**Pull request by sophieball**
#29

### CODE REVIEW

**Comment by sophieball** [comment]
I should add length or num words to the  model.

**Comment by CaptainEmerson** [comment]
I can't quite tell; do you want my reivew?

**Comment by sophieball** [comment]
> I can't quite tell; do you want my reivew?

Sorry. Now you can review =)

File: main/politeness_logi.R [hunk 1]
```diff
@@ -15,14 +15,17 @@ system2("src/convo_politeness",
...
+sapply(dat, class)
 m_pol <- glm(label ~
-              log(1 + HASHEDGE)
+              log(1+length)
+            + log(1 + HASHEDGE)
             + (Please > 0)
             + (Please_start > 0)
             + (Indirect_.btw. > 0)
-            + Hedges
+            #+ Hedges
```
Comment (CaptainEmerson) [inline main/politeness_logi.R:28]: add back in?

Comment (sophieball) [reply main/politeness_logi.R:28]: I think it should be removed. According to their paper, `HASHEDGE` is a measure, but they didn't mention `Hedges`. Also, looking at the words marked for the category (Hedges), they are not hedge words. They seem to be words that appear around hedge words.

Comment (CaptainEmerson) [reply main/politeness_logi.R]: Okey dokey. Probably add a comment to the code then.

**Pull request merged**